### DISCUSSION TITLE
Filebeat.yml  configuration

### DISCUSSION THREAD

**Discussion by davidcerve7** [Security-Onion-Solutions/securityonion#4417] [Unsupported Versions]
Hello All, 

I am interested in configuring one of the  Filebeat modules under the logs section in Kibana. I'm following the instructions provided and having a difficult time trying to modify the filebeat.yml as anything I change gets reverted back to the default configuration.

The path to the file that I'm trying to modify is /opt/so/conf/filebeat/etc/filebeat.yml
Am I configuring the correct .yml file? There's also /opt/so/saltstack/default/salt/filebeat/etc/filebeat.yml location. But the documentation mentions files under default should not be modified. I did read through this post #1714 which mentions copying over the YAML from the default to the local folder, is this still the correct way to go about this? 

Thank you, 

Security Onion Version: 2.3.5
Installation: Standalone 
Install Source: Security Onion ISO

**Comment by Git-Me-Some-Hub** [comment]
I would love to know this as well. I would really like to get Office 365 Logs ingested. I posted the question and got a few more bullet points to hit, but I am still trying to find correct wat to config. 
https://github.com/Security-Onion-Solutions/securityonion/discussions/4170

1. I have uploaded the modules.d / module folder into: /opt/so/conf/filebeat/etc
2. Entered config information in, and removed disabled.  /opt/so/conf/filebeat/etc/modules.d/o365.yml
3. Also entered config into: /opt/so/conf/filebeat/etc/filebeat.yml
4. and if it mattered also copied down the default file to the local directory: cp /opt/so/saltstack/default/salt/filebeat/init.sls /opt/so/saltstack/local/salt/filebeat/

Does not seem to be working as of now.

**Reply by Git-Me-Some-Hub** [reply]
I ended up using a separate machine that I could install Filebeats on, outside of SecurityOnion. That way I had total control. I installed the o365 module, cef, and Cisco so far.  Not the best workaround, but as they just announced and posted below to https://github.com/Security-Onion-Solutions/securityonion/discussions/4434 You just need to wait a bit to get it built in. I am taking the band-aid approach and waiting that release.

**Reply by davidcerve7** [reply]
I did the same. I'm looking forward to the next update. Thanks!

**Comment by TOoSmOotH** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/4434
